src/components/LatestRatings.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { gql } from 'apollo-boost';
+import { Query } from 'react-apollo';
+
+const Hero = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  min-height: calc(100vh - 7.375rem);
+  text-align: center;
+`;
+
+const GET_LATEST_RATINGS_QUERY = gql`
+  query GET_LATEST_RATINGS_QUERY($max: Int) {
+    getLatestRatings(max: $max) {
+      _id
+      author
+      caller
+      callType
+      message
+      score
+      createdAt
+      number {
+        number
+      }
+    }
+  }
+`;
+
+function Comment({ rating }) {
+  return (
+    <li
+      style={{
+        backgroundColor: '#e8f5e9',
+        margin: '1rem',
+        padding: '1rem',
+      }}
```

Comment:
Instead of inline styles we should use `styled-components`, it's already imported. If you need help with that, feel free to message me.